extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/HudiCommands.scala

```diff
@@ -138,5 +150,7 @@ object HudiCommands {
     CreateHoodieTableAsSelectCommand,
     CreateHoodieTableLikeCommand,
     DropHoodieTableCommand,
-    TruncateHoodieTableCommand)
+    TruncateHoodieTableCommand,
+    CompactionHoodieTableCommand,
+    CompactionShowHoodieTableCommand)
```

Comment:
Sort the list in alphabet order for better maintainability.

Reply:
Done